ferry.go

```diff
@@ -721,7 +765,14 @@ func (f *Ferry) FlushBinlogAndStopStreaming() {
 		}
 	}
 
-	f.BinlogStreamer.FlushAndStop()
+	f.SourceBinlogStreamer.FlushAndStop()
+}
+
+func (f *Ferry) Stop() {
+	if !f.Config.SkipTargetVerification {
+		f.TargetBinlogStreamer.FlushAndStop()
+		f.targetVerifierWg.Wait()
+	}
```

Comment:
Not sure about the purpose of this. Why do we need both this and the calls to `r.Ferry.TargetVerifier.CutoverCompleted.Set(true)`

Can they all be rolled up into one thing?